src/main/java/org/jenkinsci/plugins/stashNotifier/StashNotifier.java

```diff
@@ -660,6 +688,24 @@ public boolean configure(
 		}
 	}
 
+	@Extension
+	public static class BuildResultTokenMacro extends DataBoundTokenMacro {
```

Comment:
@kutsal Macro extension is still present here, but is gone from fields description. Is it still used?